chain/exchange/protocol.go

```diff
@@ -13,9 +13,17 @@ import (
 	"github.com/filecoin-project/lotus/chain/types"
 )
 
-var log = logging.Logger("blocksync")
+var log = logging.Logger("chainxchg")
 
-const BlockSyncProtocolID = "/fil/sync/blk/0.0.1"
+const (
+	// BlockSyncProtocolID is the protocol ID of the former blocksync protocol.
+	// Deprecated.
+	BlockSyncProtocolID = "/fil/sync/blk/0.0.1"
+
+	// ChainExchangeProtocolID is the protocol ID of the chain exchange
+	// protocol.
+	ChainExchangeProtocolID = "/fil/chain/xchg/0.0.1"
+)
```

Comment:
For consistency, I had to rename the p2p protocol. It's totally fine since we register the stream handler under both protocol IDs. And when opening an outbound stream, we try ChainExchange first, falling back to BlockSync. Before releasing v1, we can probably get rid of the deprecated one entirely.

Reply:
Can you open an issue so we can track this?

Reply:
Done: https://github.com/filecoin-project/lotus/issues/3737.